third_party/dependency_analyzer/src/test/BUILD

```diff
@@ -4,8 +4,8 @@ load("//scala:scala.bzl", "scala_junit_test", "scala_test")
 
 common_jvm_flags = [
     "-Dplugin.jar.location=$(location //third_party/dependency_analyzer/src/main:dependency_analyzer)",
-    "-Dscala.library.location=$(location //external:io_bazel_rules_scala/dependency/scala/scala_library)",
-    "-Dscala.reflect.location=$(location //external:io_bazel_rules_scala/dependency/scala/scala_reflect)",
+    "-Dscala.library.location=$(rootpath //external:io_bazel_rules_scala/dependency/scala/scala_library)",
```

Comment:
don't remember the difference between location and rootpath. Do you remember?

Reply:
I don't think they are good for runtime information, but as we use them there's some info on them:

https://docs.bazel.build/versions/master/be/make-variables.html#predefined_label_variables

and 

https://github.com/bazelbuild/bazel/issues/2475#issuecomment-339318016